frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+      name: key,
+      percentage: (count / allTests.length) * 100,
+    }))
+  }
+  const allTestsStatus = prepareAllTestsStatus()
+  const pageLayout = usePageLayout({
+    header: <Header />,
+    footer: <PageLayout.Footer />,
+  })
+  return (
+    <PageLayout {...pageLayout}>
+      <PageLayout.Section>
+        <BarWrapper>
+          <Text type="text xl">All Tests Kinds</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsKinds.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <BarWrapper>
+          <Text type="text xl">All Tests Status</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsStatus.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <Heading>
+          <Text type="text xl">List of all tests ({allTests.length})</Text>
+        </Heading>
+        <Table
+          variant="default"
+          columns={{
+            suite: {
+              header: 'Suite',
+              width: 220,
+              Cell: ({ data }) => {
+                return (
+                  <>
+                    <Text type="text s">
+                      {data.id.split('/')[0]}
+                      <br />
+                    </Text>
+                    <Link
+                      to={data.id.split('/')[1]}
+                      appearance="system"
+                      icon="link"
+                      className="u-text--xsmall"
+                    >
+                      {data.id.split('/')[1]}
+                    </Link>
+                  </>
+                )
+              },
+            },
+            function: {
+              header: 'Function',
+              Cell: ({ data }) => {
+                return <Text type="text s">{data.functionName}</Text>
+              },
+            },
+            repository: {
+              header: 'Repository',
+
+              Cell: ({ data }) => {
+                return (
+                  <Link
+                    to={data.repository}
+                    appearance="system"
+                    className="u-text--xsmall"
+                  >
+                    {data.repository}
+                  </Link>
+                )
+              },
+            },
+            kinds: {
+              header: 'Kinds (tags)',
+
+              Cell: ({ data }) => {
+                return (
+                  <Link
+                    to={data.kind}
+                    appearance="system"
+                    className="u-text--xsmall"
+                  >
+                    {data.kind}
+                  </Link>
+                )
+              },
+            },
+            behaviors: {
+              header: 'Behaviors',
+
```

Comment:
This is not formatted with prettier

Reply:
what could lead to that ? 
my .prettierrc :
![image](https://user-images.githubusercontent.com/85948457/143399452-2d357e7b-64c6-40d9-9c80-efb3d2cd5cb9.png)

my Vs code settings:
![image](https://user-images.githubusercontent.com/85948457/143399539-c0b33ef8-0ed5-48f1-89ea-0b10afbfa78c.png)

Reply:
Not sure, I'll Resolve this until we found the issue.